### DISCUSSION TITLE
一个随Star数动态变化的GitHub仓库 - 逸風亭

### DISCUSSION THREAD

**Discussion by windshadow233** [windshadow233/BlogStorage#133] [Announcements]
https://blog.fyz666.xyz/blog/9506/

**Comment by iBug** [comment]
过来凑个热闹，我的那篇博客在

中文：<https://ibug.io/p/41-cn>
English: <https://ibug.io/p/41>

以及欢迎来围观[我的仓库](https://github.com/iBug/This-Repo-Has-0-Stars)～
